browser/src/Services/Commands/GlobalCommands.ts

```diff
@@ -45,6 +50,78 @@ export const activate = (
     const popupMenuPrevious = popupMenuCommand(() => menuManager.previousMenuItem())
     const popupMenuSelect = popupMenuCommand(() => menuManager.selectMenuItem())
 
+    const gotoNextError = async () => {
```

Comment:
Could these functions be move into their own files in the `Diagnostics` directory maybe `Diagnostics/navigateErrors.ts` so the global commands isn't where they are implemented and just import them here?

Reply:
Thanks for the feedback! I moved this to `browser/src/Services/Diagnostics/navigateErrors.ts`, but that might be confusing since there's already a `browser/src/Services/Diagnostics.ts`. I can move that file inside the new `browser/src/Services/Diagnostics` directory if you want.

Reply:
we could make the already existing `Diagnostics.ts => browser/src/Services/Diagnostics/index.ts` that way if there are future helpers etc. they can go in the same dir plus you wont have to change any imports since as index all the imports of the file should work as is